crates/sui-indexer-alt-e2e-tests/src/lib.rs

```diff
@@ -192,7 +192,8 @@ impl FullCluster {
         self.offchain.latest_checkpoint().await
     }
 
-    /// Waits until the indexer has caught up to the given checkpoint, or the timeout is reached.
+    /// Waits until the indexer has caught up to the given `checkpoint`, or the `timeout` is
```

Comment:
```suggestion
    /// Waits until the indexer has committed up to the given `checkpoint`, or the `timeout` is
```

Something like that, given the language for `wait_for_pruner`?

We could also have tests around `reader_lo`